pkg/shell/superuser.jsx

```diff
@@ -108,7 +110,16 @@ const UnlockDialog = ({ proxy, host }) => {
 
     function init() {
         return proxy.Stop().always(() => {
-            start("sudo");
+            if (proxy.Methods) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220810-164203-49e829c5-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)